controllers/routing/routing_reconcile_resources.go

```diff
@@ -73,6 +82,15 @@ func (r *PlatformRoutingController) exportService(ctx context.Context, target *u
 
 	targetKey := k8stypes.NamespacedName{Namespace: target.GetNamespace(), Name: target.GetName()}
 
+	for _, routeType := range spi.AllRouteTypes() {
+		if !spi.ContainsRouteType(exportModes, routeType) {
+			err := r.deleteOwnedResources(ctx, target, routingResourceGVKs(routeType))
+			if err != nil {
+				r.log.Error(err, "could not delete owned resources", "target", targetKey, "routeType", routeType)
```

Comment:
wonder if this should trigger a new reconcile attempt.. so basically push it with a multi error further up?